Makefile

```diff
@@ -18,6 +18,26 @@ clean:
   		echo "Nothing to clean."; \
 	fi
 
+all:
```

Comment:
Since there are multiple moving parts running `make all` provides and helpful text because there are different ways to “make all the thingz” :).